elements/rh-band/demo/index.html

```diff
@@ -17,21 +17,28 @@
     <script>require([
       '../rh-band.umd.js',
       '../../rh-card/rh-card.umd.js',
-      '../../rh-cta/rh-cta.umd.js'
+      '../../rh-cta/rh-cta.umd.js',
+      '../../../themes/cp-theme/cp-theme.umd.js',
     ])</script>
+    <style>
+      .pfe-band-summary {
```

Comment:
Even though this is just a demo, I think I'd prefer if classes that are not added by the component's template don't use the pfe prefix.

Reply:
That way, developers looking at the demo page don't think they should be doing the same for their custom classes

Reply:
sure thing